ui/app/components/ui/item/item.component.js

```diff
@@ -0,0 +1,141 @@
+
```

Comment:
will these items always be items in a list? if so, the code might be a little more self-documenting if we named this "list-item"